hikari/api/rest.py

```diff
@@ -2195,7 +2624,49 @@ async def create_guild_voice_channel(
         category: undefined.UndefinedOr[snowflakes.SnowflakeishOr[channels.GuildCategory]] = undefined.UNDEFINED,
         reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> channels.GuildVoiceChannel:
-        ...
+        """Create a voice channel in a guild.
+
+        Parameters
+        ----------
+        guild : hikari.snowflakes.SnowflakeishOr[hikari.guilds.PartialGuild]
+            The guild to create the channel in. This may be a
+            guild object or the ID of an existing guild.
+        name : builtins.str
+            The channels name.
+        position : hikari.undefined.UndefinedOr[builtins.int]
+            If provided, the position of the channel (relative to the
+            category, if any).
+        user_limit : hikari.undefined.UndefinedOr[builtins.int]
+            If provided, the user limit for the channel.
```

Comment:
Could be more specific around this being a limit for how many users can be in the channel.